bitnami/spark/templates/NOTES.txt

```diff
@@ -0,0 +1,37 @@
+
+** Please be patient while the chart is being deployed **
+
+1. Get the Spark master WebUI URL by running these commands:
+
+{{- if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "spark.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "spark.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "spark.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  echo http://$SERVICE_IP:{{ .Values.service.port }}
+{{- else if contains "ClusterIP" .Values.service.type }}
+  kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ template "spark.fullname" . }}-master-svc {{ default "8080" .Values.service.port }}:8080
+  echo "Visit http://127.0.0.1:{{ .Values.master.webPort }} to use your application"
+{{- end }}
+
+2. Submit an application to the cluster:
+
+To submit an application to the cluster the spark-submit script must be used. That script can be
+obtained at https://github.com/apache/spark/tree/master/bin.
+
+First, obtain the master IP, to do that the service type must be NodePort or LoadBalancer. Run the following command to obtain the master IP and submit your application:
+
+$ export MASTER_IP=$(kubectl get services | awk '/{{ include "spark.fullname" . }}/ { print $3 }')
+$ spark-submit --master spark://$MASTER_IP:{{ .Values.master.clusterPort }} --deploy-mode cluster  /path/to/application 1000
+
+To access the service IP you should use service.type=NodePort or service.type=LoadBalancer, so you can reach the cluster.
+
+
+** Please be patient while the chart is being deployed **
```

Comment:
You already set this above, do you need to set it here again?

Reply:
Right